api/managementpb/discovery.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package management;
+
+// This option is used to set the default error response for all Swagger APIs generated from this file.
+// See also header.json.
+// It has to be copy&pasted into each file.
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  responses: {
+    key: "default"
+    value: {
+      description: "An error response."
+      schema: {
+        json_schema: {
+          ref: "#/definitions/managementpbErrorResponse"
+        }
+      }
+    }
+  }
+};
+option go_package = "managementpb";
+
+import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+// DiscoverRDSEngine describes supported RDS instance engines.
+enum DiscoverRDSEngine {
+  DISCOVER_RDS_ENGINE_INVALID = 0;
+  DISCOVER_RDS_MYSQL = 1;
```

Comment:
So we don't accept RDS PostgreSQL databases for now?

Reply:
We are reducing a scope for 2.2.0 / AWS Re:Invent. I don't know when we plan to add PostgreSQL and Aurora variants. I does not look hard to implement, but it will require more QA time – and we still don't have a configured account.